azure-local/concepts/compare-windows-server.md

```diff
@@ -69,60 +69,65 @@ The following table compares the workloads and benefits of Azure Local and Windo
 
 | Attribute | Azure Local | Windows Server |
 | ------------- | ------------------- | ----------------------- |
-| Azure Kubernetes Service (AKS)| Yes | Yes |
+| Azure Kubernetes Service (AKS)| Yes | Yes, but sunset <sup>1</sup> |
 | Azure Arc-Enabled PaaS Services | Yes | Yes |
-| Windows Server 2022 Azure Edition | Yes | No|
+| Windows Server 2022 Azure Edition | Yes | No |
+| Windows Server 2025 Azure Edition | Yes | No |
```

Comment:
added release, as readers had the impression WS 2025 is not yet available as Azure Edition. But it is available from Azure marketplace image